src/main/webapp/app/tasks/local-task.service.spec.ts

```diff
@@ -52,4 +52,13 @@ describe('LocalTaskService', () => {
     expect(localStorage.getItem).toHaveBeenCalled();
     expect(localStorage.setItem).toHaveBeenCalled();
   });
+  
+  it('should update task in local storage', () => {
+    // when
+    taskService.update({ id: id, name: 'Buy a kilo of Ritter Sport chocolate' });
+
+    // then
+    expect(localStorage.getItem).toHaveBeenCalled();
```

Comment:
as the expects are the same as above this can't really test the functionality of this single method